CMakeLists.txt

```
@@ -26,6 +26,7 @@ set(PROJECT_VERSION "2.3.0")
 if(NOT WIN32)
   string(APPEND CMAKE_CXX_FLAGS " -Wno-error")
 endif()
+string(APPEND CMAKE_CXX_FLAGS " -Werror=unused-variable")
```

```suggestion
  string(APPEND CMAKE_CXX_FLAGS " -Wno-error")
  string(APPEND CMAKE_CXX_FLAGS " -Werror=unused-variable")
endif()
```
Restore for linux-only, as it breaks Windows CI